docs/cloud.rst

```diff
@@ -73,16 +74,16 @@ a GitHub Team in the ``pangeo-data`` organization.
 You must accept this invitation to gain access to Pangeo Cloud resources.
 
 
-Clusters
---------
+Hubs
+----
 
-Pangeo Cloud currently comprises two different computing clusters:
+Pangeo Cloud currently comprises two different JupyterHubs:
 
-- `us-central1-b.gcp.pangeo.io <https://us-central1-b.gcp.pangeo.io/>`_: A cluster in
+- `us-central1-b.gcp.pangeo.io <https://us-central1-b.gcp.pangeo.io/>`_: A hub in
   Google Cloud Platform.
-- `aws-uswest2.pangeo.io <https://aws-uswest2.pangeo.io>`_: A cluster in AWS.
+- `aws-uswest2.pangeo.io <https://aws-uswest2.pangeo.io>`_: A hub in AWS.
 
-We recommend choosing a cluster based on which data you want to access.
+We recommend choosing a hub based on which data you want to access.
```

Comment:
Just looking at this PR, I don't see a link to the information about which data are in the hubs.

Reply:
There are no data "in" the hubs. There are different public datasets in different cloud providers and regions. I will make a clarification.